Makefile

```diff
@@ -83,8 +85,15 @@ help: ## Show this help screen
 docs/packages/%.html: %.go
 	mkdir -p $(dir $@)
 	docgo -outdir $(dir $@) $^
+	addlicense -c "Red Hat, Inc" -l "apache" -v $@
 
-godoc: ${DOCFILES}
+godoc: export GO111MODULE=off
+godoc: install_docgo install_addlicense ${DOCFILES}
 
-openapi-check:
-	./check_openapi.sh
+install_docgo: export GO111MODULE=off
+install_docgo:
+	[[ `command -v docgo` ]] || GO111MODULE=off go get -u github.com/dhconnelly/docgo
+
+install_addlicense: export GO111MODULE=off
```

Comment:
It's will be run before addlicense by makefile's magic? Did I get it correctly?

Reply:
Makefile's magic will export that environment variable to the execution of the rule `install_addlicense`, so the execution of that rule will be covered by this env var definition.

It answers your previous comment. I remove `GO111MODULE` from one side and put in a different place. It has more sense with `godoc` rule, that will avoid command `docgo` to update `go.sum` and `go.mod`.

Reply:
Thanks